src/Compilers/Core/Portable/DiagnosticAnalyzer/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -53,6 +53,10 @@ public ShadowCopyAnalyzerAssemblyLoader(string baseDirectory = null)
 
         private void DeleteLeftoverDirectories()
         {
+            // Avoid first chance exception
+            if (!Directory.Exists(_baseDirectory))
+                return;
+
```

Comment:
can we remove the try/catch below with this change?

Reply:
Nope, as there's still a race that we can lose.